vignettes/strayr.Rmd

```diff
@@ -0,0 +1,154 @@
+---
+title: "StrayR"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{StrayR}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+## Under Construction
+
+
+## Using ABS structures
+
+Loading the package will lazily load the structures listed above. Call
+them with their name:
```

Comment:
The structures are no longer "listed above", maybe a link to the reference works?

```suggestion
Loading the package will lazily load a number of structures, a full list is available in the [reference](../reference/index.html)
```